multichain-aggregator/multichain-aggregator-server/src/server.rs

```diff
@@ -65,11 +65,21 @@ pub async fn run(settings: Settings) -> Result<(), anyhow::Error> {
         .fetch_all()
         .await?
         .into_iter()
-        .map(|(id, chain)| (id as i64, chain).into())
+        .filter_map(|(id, chain)| {
+            let id = id.parse::<i64>().ok()?;
```

Comment:
the decision was to ignore the chain ids with prefixes for now?

Reply:
Yes, we want to skip chains with non-integer chain ids